src/functions/proximalMaps.jl

```diff
@@ -267,12 +265,12 @@ function prox_TV2(M::PowerManifold{N,T}, λ, x, p::Int=1) where {N,T}
   maxInd = [last(R).I...]
   y = copy(x)
   for k in 1:d # for all directions
+    ek = CartesianIndex(ntuple(i  ->  (i==k) ? 1 : 0, d) ) #k th unit vector
     for l in 0:1
       for i in R # iterate over all pixel
         if (i[k] % 3) == l
-          I = [i.I...] # array of index
-          JForward = I .+ 1 .* (1:d .== k) #i + e_k
-          JBackward = I .+ 1 .* (1:d .== k) # i - e_k
+          JForward = i.I .+ ek.I #i + e_k
+          JBackward = i.I .- ek.I # i - e_k
```

Comment:
I guess this was a bug?

Reply:
Both `JForward` and `JBackward` had the same code (with addition)..

Reply:
Oh, yes, thanks for noticing; I actually have not yet used this function in Julia (just transcribed it from my Matlab code) – so yes this is a copy paste error.